README.md

```diff
@@ -25,19 +41,31 @@ git clone --recursive https://github.com/chloroExtractorTeam/chloroExtractor
 ```
 
 ### Docker
+[![Docker Automated build](https://img.shields.io/docker/automated/chloroextractorteam/chloroextractor.svg?style=plastic)](https://hub.docker.com/r/chloroextractorteam/chloroextractor/)
+[![Docker Build Status](https://img.shields.io/docker/build/chloroextractorteam/chloroextractor.svg?style=plastic)](https://hub.docker.com/r/chloroextractorteam/chloroextractor/)
 
-chloroExtractor is also available as a docker image.
-TODO @greatfireball add instructions
+Our chloroExtractor is also available as a docker image.
+Running chloroExtractor using that image requires the installation of docker and the permission to execute the `docker` commands.
+The data are mapped into the container as a volumne under `/data`.
+
+```shell
+docker pull chloroextractorteam/chloroextractor
+docker run -v /location-of-input-data:/data --rm chloroextractorteam/chloroextractor -1 first_read.fq -2 second_read.fa [other options]
+```
```

Comment:
Great, short note about where the output files will end up, maybe with a warning that they might belong to root would be a good thing

Reply:
I added some more information plus warnings and link to https://docker.com in commit cf1f75d1ab5206c62294677dd8460876c48da978

Reply:
:+1: